core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -454,6 +454,9 @@ public String getAbsoluteRemoteFs() {
         }
         @Override public Integer call() {
             Channel c = Channel.current();
+            if (c == null) {
+                return -1;
```

Comment:
I used -1, because other counts in this class return "-1" in exceptional cases. E.g.

```java
  public int getClassLoadingPrefetchCacheCount() throws IOException, InterruptedException {
        if (!channel.remoteCapability.supportsPrefetch()) {
            return -1;
        }
        return channel.call(new LoadingPrefetchCacheCount());
    }
```